packages/node-cli/src/utils/SdkOperationResult.js

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
May I know the reason why there is a need to create SdkOperationResult instead of just using OperationResult?

Reply:
It's a convinient way to kind of "type" via jsdocs the return type of an executed commad via sdkExecutor.
If you open the code in an editor you'll see that there is some "code completion" available when using the recently created methods on AuthenticationUtils.js
Also we can add some useful functionality as the .isSuccess() method, instead of comparing `operationResult.status === SdkOperationResultUtils.STATUS.SUCCESS`. There is a comment about it on the `src\utils\SdkOperationResultUtils.js`

Reply:
Understood.